components/builder-api/src/http/handlers.rs

```diff
@@ -642,7 +643,30 @@ pub fn project_jobs(req: &mut Request) -> IronResult<Response> {
     }
     let mut conn = Broker::connect().unwrap();
     match conn.route::<ProjectJobsGet, ProjectJobsGetResponse>(&jobs_get) {
-        Ok(jobs) => Ok(render_json(status::Ok, &jobs)),
+        Ok(response) => {
+            let list: Vec<serde_json::Value> = response
+                .get_jobs()
+                .iter()
+                .map(|job| if job.get_package_ident().fully_qualified() {
```

Comment:
I think you are constraining to fully qualified idents because that would effectively limit this to completed jobs? If thats the case, making this `job.get_state() == JobState::Complete` might be more clear.

Reply:
Oddly enough, that's not the reason I constrained it to fully qualified idents, but after thinking your comment over, I think constraining it to only completed jobs makes more sense (it's possible for an ident to be fully qualified and still have that job fail).  Thanks for the suggestion, will do.